### PR TITLE
[WIP] Reused code of rocket-container.

### PR DESCRIPTION
**Pull request by jainarunarunjain**
@aka-jain Need some help ,
In this ng-include is not showing any content.

### CODE REVIEW

**Comment by aka-jain** [comment]
@jainarunarunjain will check it.

File: frontend/src/views/web/about-us.html [hunk 1]
```diff
@@ -14,22 +14,7 @@ <h4 class="text-dark-black align-center"><strong>About Us</strong></h4>
...
-                    </div>
-                    <div class="cloud-2">
-                        <img src="dist/images/cloud-2.png">
-                    </div>
-                    <div class="cloud-3">
-                        <img src="dist/images/cloud-1.png">
-                    </div>
-                </div>
-            </div>
+            <div ng-include="partials/rocket-container.html"></div>
```
Comment (aka-jain) [inline frontend/src/views/web/about-us.html]: Chang this path to 'dist/views/web/partials/rocket-container.html'

Comment (jainarunarunjain) [inline frontend/src/views/web/about-us.html]: @aka-jain Sure.

**Comment by aka-jain** [comment]
LGTM. Merging it.

**Comment by jainarunarunjain** [comment]
@aka-jain @spyshiv Please Review.

**Pull request merged**